main.py

```diff
@@ -24,17 +28,44 @@ def main():
         test = MultipleFeatures_Test()
     else:
         test = BrownNiekum()
+    return test
+
+
+def main():
```

Comment:
Instead of making main() the staging area for all of the testing code, can we make a separate function for running the tests?